pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go

```diff
@@ -1170,21 +1145,22 @@ func TestPreempt(t *testing.T) {
 			expectedPods:   []string{"p1.1", "p1.2"},
 		},
 		{
-			name: "One scheduler extender allows only machine1, but it is not interested in given pod, otherwise node1 would have been chosen",
+			name: "One scheduler extender allows only machine1, but it is not interested in given pod, otherwise machine1 would have been chosen",
 			pod:  st.MakePod().Name("p").UID("p").Priority(highPriority).Req(veryLargeRes).PreemptionPolicy(v1.PreemptLowerPriority).Obj(),
 			pods: []*v1.Pod{
-				st.MakePod().Name("p1.1").UID("p1.1").Node("node1").Priority(midPriority).Req(smallRes).Obj(),
-				st.MakePod().Name("p1.2").UID("p1.2").Node("node1").Priority(lowPriority).Req(smallRes).Obj(),
-				st.MakePod().Name("p2.1").UID("p2.1").Node("node2").Priority(midPriority).Req(largeRes).Obj(),
+				st.MakePod().Name("p1.1").UID("p1.1").Node("machine1").Priority(midPriority).Req(smallRes).Obj(),
+				st.MakePod().Name("p1.2").UID("p1.2").Node("machine1").Priority(lowPriority).Req(smallRes).Obj(),
+				st.MakePod().Name("p2.1").UID("p2.1").Node("machine2").Priority(midPriority).Req(largeRes).Obj(),
```

Comment:
I reverted the "node" to "machine" cause the extender used here is still using "machine" instead of "node", this will make the this testcase invalid without the change here.

https://github.com/kubernetes/kubernetes/blob/865cbf0bdfb4ae0d6cb5029ffacea127cd42e89c/pkg/scheduler/testing/fake_extender.go#L62-L67

As there are still other occurrences of "machine" in other testcase. I will propose another PR to address it instead of mess up this PR.

Reply:
cc @Huang-Wei

Reply:
see: https://github.com/kubernetes/kubernetes/pull/92819